beancount/parser/Makefile

```diff
@@ -0,0 +1,4 @@
+PYTHON := python-3.7
```

Comment:
These file should be removed and its contents inserted in the Makefile at the root.

Reply:
I didn't do this because this Makefile is mostly an hack that work locally, but making it work reliably on a wide range of systems would be more complex. I don't know of a reliable way to get the right argument to `pkg-config` from a python executable path that is accepted by the project Makefile. I'll see if I can come up with something that does not look too complex.